### PR TITLE
chore: Update dependencies and remove legacy code

### PR DESCRIPTION
**Pull request by pksunkara**

### CODE REVIEW

File: test/refract-test.coffee [hunk 1]
```diff
@@ -255,8 +259,8 @@ describe('Transformations • Refract', ->
           assert.equal(resource.parameters[1].key, 'arg')
         )
 
-        it('second parameter isn\'t required', ->
-          assert.isFalse(resource.parameters[1].required)
+        it('second parameter is required', ->
+          assert.isTrue(resource.parameters[1].required)
```
Comment (pksunkara) [inline test/refract-test.coffee:263]: The swagger fixture was changed in this case.

File: test/refract-test.coffee [hunk 2]
```diff
@@ -156,10 +159,6 @@ describe('Transformations • Refract', ->
         it('has actionDescription', ->
           assert.isOk(resource.actionDescription)
         )
-
-        it('has actionRelation', ->
-          assert.isOk(resource.actionRelation)
-        )
```
Comment (pksunkara) [inline test/refract-test.coffee:162]: Swagger doesn't have `action.relations`. We fixed that bug.

**Comment by pksunkara** [comment]
@Baggz Can I get a review on this? thanks.

File: test/refract-test.coffee (hunk 2 above)
Comment (Baggz) [reply test/refract-test.coffee:162]: :+1:

**Comment by kuba-kubula** [comment]
Lodash 4.x does not (officially) support NodeJS 0.10, 0.12... Which is going to be a breaking change for us, I'd say.

**Comment by pksunkara** [comment]
We are testing this repo on node 0.10 as shown [here](https://github.com/apiaryio/metamorphoses/blob/master/circle.yml#L3). Shouldn't this be okay with npm shrink-wrap?

**Comment by pksunkara** [comment]
Relevant PR in apiary. https://github.com/apiaryio/apiary/pull/4086

**Pull request merged**